src/lib.rs

```diff
@@ -1579,7 +1579,27 @@ impl Bump {
     ///     assert_eq!(chunk[2].assume_init(), b'a');
     /// }
     /// ```
+    #[inline(always)]
     pub fn iter_allocated_chunks(&mut self) -> ChunkIter<'_> {
+        // SAFE: `self` is borrowed mutably.
+        unsafe { self.iter_allocated_chunks_unchecked() }
```

Comment:
Removed a little bit of newly-duplicated code, but I can revert this section if you want to avoid a new `unsafe` usage (even if trivial) for not much benefit.